cluster/saltbase/salt/e2e-image-puller/e2e-image-puller.manifest

```diff
@@ -35,6 +35,14 @@ spec:
       name: socket
     - mountPath: /usr/bin/docker
       name: docker
+  # Add a container that runs a health-check
+  - name: nethealth-check
+    resources:
+      requests:
+        cpu: 100m
```

Comment:
Ah, got it, I used the same resource limits as the image-puller container. This is a short lived container, max 30 seconds; it should vacate its reserved CPU before other pods are scheduled.